packages/web/assets/demo.css

```diff
@@ -50,12 +50,6 @@
   white-space: normal;
 }
 
-.docs-FormFieldGrid {
-  display: grid;
-  grid-template-columns: 25rem;
-  gap: 1.5rem;
-}
-
```

Comment:
Does @crishpeen remember what the size limit was for? I took a quick glance over over a few form field demos and did not see any major difference or reason…

Reply:
Not really, if the form-field look good, we probably don't need it anymore